content/rs/release-notes/rs-6-2-10-february-2022.md

```diff
@@ -97,10 +98,23 @@ For help upgrading a module, see [Add a module to a cluster](https://docs.redis.
          
     - For Active-Passive (Replica Of) databases: use the admin console to verify that the destination syncer has the correct certificate for the source proxy (DMC).  For details, see [Configure TLS for Replica Of](https://docs.redis.com/latest/rs/administering/creating-databases/create-active-passive/#configuring-tls-for-replica-of-traffic-on-the-destination-database).
 
-## Resolved issues (build 100)
+### Issues resolved in build 96
+
+- RS67133 - An issue in Redis Enterprise Software affected replication in replica databases using RedisGraph, RediSearch, and RedisGears in specific scenarios.  The problem appeared when importing an RDB file or while synchronizing target Active-Passive (ReplicaOf) databases. 
+
+    This issue is fixed in Redis Enterprise Software v6.2.10-96 and RedisGraph v2.8.11.  We recommend upgrading to these versions at your earliest opportunity.  (Failure to upgrade can lead to data loss.)  
+    
+    Once the upgrades are complete, secondary shards might need to be restarted.  You can use `rlutil` to restart secondary shards:
```

Comment:
@lanceleonard why did we keep the might?  How is the users supposed to know?  let's always tell him to restart